<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LERMA</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1902</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="viewSW">
      <property name="font">
       <font>
        <family>Gill Sans MT</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QGroupBox" name="signUpGB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Gill Sans MT</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font:rgb(255, 255, 255)</string>
          </property>
          <property name="title">
           <string>Sign Up</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLineEdit" name="newUserLE">
             <property name="font">
              <font>
               <family>Gill Sans MT</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="placeholderText">
              <string>User</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="emailLE">
             <property name="font">
              <font>
               <family>Gill Sans MT</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="placeholderText">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="newPasswordLE">
             <property name="font">
              <font>
               <family>Gill Sans MT</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="signUpPB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Gill Sans MT</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Sign Up</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="loginGB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Gill Sans MT</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="title">
           <string>Login</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QPushButton" name="loginPB">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Gill Sans MT</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="usernameLE">
             <property name="font">
              <font>
               <family>Gill Sans MT</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="placeholderText">
              <string>User</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="passwordLE">
             <property name="font">
              <font>
               <family>Gill Sans MT</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Password</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="tittleLBL">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Bahnschrift Light</family>
            <pointsize>31</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>LERMA</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="2" column="0">
         <widget class="QScrollArea" name="scrollAreaRecommended">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>230</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>985</width>
             <height>188</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <layout class="QGridLayout" name="recommendedGrid"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>985</width>
             <height>189</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <layout class="QGridLayout" name="productsGrid">
              <property name="spacing">
               <number>7</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="departmentsGB">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Departments</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="0">
            <widget class="QLineEdit" name="searchLE">
             <property name="placeholderText">
              <string>Product Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="departmentCB">
             <item>
              <property name="text">
               <string>All Departments</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Food &amp; Drinks</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Books</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Electronics</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Home &amp; Kitchen</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sport &amp; Outdoors</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="sortCB">
             <item>
              <property name="text">
               <string>Sort...</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ascending</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Descending</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
